src/pages/my-lessons/MyLessons.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react'
+import Box from '@mui/material/Box'
+
+import Tab from '~/components/tab/Tab'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+import { styles } from '~/pages/my-lessons/MyLessons.styles'
+
+const groupedLessonsDataMock = [
+  {
```

Comment:
We discussed that there will be only two tabs in the page (lessons and courses or grouped lessons)

Reply:
There are, on the left - grouped lessons and on the right - lessons

Reply:
fixed